hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -703,8 +703,8 @@ public LeaseKeyInfo recoverFilePrepare(final String pathStr, boolean force) thro
     incrementCounter(Statistic.INVOCATION_RECOVER_FILE_PREPARE, 1);
 
     try {
-      return ozoneClient.getProxy().getOzoneManagerClient().recoverLease(
-          volume.getName(), bucket.getName(), pathStr, force);
+      RpcClient rpcClient = (RpcClient) ozoneClient.getProxy();
```

Comment:
Why add the cast if the methods like `recoverLease` are also added to the `ClientProtocol` interface?